Makefile

```diff
@@ -321,34 +352,38 @@ mec-tools/mec_encrypt: mec-tools/Makefile
 	git submodule update
 	make -C mec-tools
 
+mec-tools/mec_csum_outer: mec-tools/Makefile
+	git submodule update
+	make -C mec-tools
+
 # using function calls to build rules with actions is kind of a hack,
 # which is why these are all on oneline.
 
 # Insert the new firmware into the FL2 file
-# $1 = encoded EC firmware
+# $1 = repacked (csums fixed and optionally re-encrypted) EC firmware
 # $2 = FL2 filename
 define rule_fl2_patch
-    $(2): $(1) ; ./scripts/slice.insert $(1).slice $(1) $(2)
+    $(2): $(1) ./scripts/slice.insert ; ./scripts/slice.insert $(1).slice $(1) $(2)
```

Comment:
Yes, technically this rule does depend on slice.insert, however changes to the slice.insert script (and similarly, the copyFL2 script below) should not change the contents of the resultant output.

(Conversely, there would not normally be any changes to these scripts, so no rebuilds would be triggered)

Can you expand a little on what your reasoning was?